setup.cfg

```diff
@@ -16,6 +16,7 @@ python_requires = >=3.8
 install_requires =
     curated-tokenizers>=0.0.7,<0.1.0
     huggingface-hub>=0.14
+    tokenizers
```

Comment:
No pins on this requirement?

Reply:
Set the lower version to the current version.